src/utils/dataFormatters/dataFormatters.ts

```diff
@@ -52,7 +52,12 @@ export const formatMsToUptime = (ms?: number) => {
     return ms && formatUptime(ms / 1000);
 };
 
-export const formatStorageValues = (value?: number, total?: number, size?: BytesSizes) => {
+export const formatStorageValues = (
```

Comment:
please add jest tests to this function that covers corner cases like infinite loop in while